games-action/polymc/polymc-1.1.1.ebuild

```
@@ -0,0 +1,108 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake java-pkg-2 optfeature xdg
+
+HOMEPAGE="https://polymc.org/"
+DESCRIPTION="A custom, open source Minecraft launcher"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="
+		https://github.com/PolyMC/PolyMC
+		https://github.com/MultiMC/libnbtplusplus
+		https://github.com/stachenov/quazip
+	"
+
+	# Include all submodules
+	EGIT_SUBMODULES=( '*' )
+else
+	MY_PN="PolyMC"
+
+	# Let's use the vendored tarball to avoid dealing with the submodules directly
+	SRC_URI="
+		https://github.com/PolyMC/PolyMC/releases/download/${PV}/${MY_PN}-${PV}.tar.gz -> ${P}.tar.gz
+	"
+
+	# The PolyMC's files are unpacked to ${WORKDIR}/PolyMC-${PV}
+	S="${WORKDIR}/${MY_PN}-${PV}"
+
+	KEYWORDS="~amd64"
+fi
+
+# Apache-2.0 for MultiMC (PolyMC is forked from it)
+# GPL-3 for PolyMC
+# LGPL-3 for libnbtplusplus
+# LGPL-2.1 with linking exception for Quazip
+# See the rest of PolyMC's libraries at https://github.com/PolyMC/PolyMC/tree/develop/libraries
+LICENSE="Apache-2.0 Boost-1.0 BSD BSD-2 GPL-2+ GPL-3 LGPL-3 LGPL-2.1-with-linking-exception OFL-1.1 MIT"
+
+SLOT="0"
+
+IUSE="debug"
+
+MIN_QT="5.6.0"
+
+QT_DEPS="
+	>=dev-qt/qtcore-${MIN_QT}:5
+	>=dev-qt/qttest-${MIN_QT}:5
+	>=dev-qt/qtconcurrent-${MIN_QT}:5
+	>=dev-qt/qtgui-${MIN_QT}:5
+	>=dev-qt/qtnetwork-${MIN_QT}:5
+	>=dev-qt/qtwidgets-${MIN_QT}:5
+	>=dev-qt/qtxml-${MIN_QT}:5
+"
+
+DEPEND="
+	${QT_DEPS}
+	>=virtual/jdk-1.8.0:*
+	media-libs/libglvnd
+	sys-libs/zlib
```

```suggestion
	media-libs/libglvnd
	sys-libs/zlib
	>=virtual/jdk-1.8.0:*
```
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Sorting_dependencies